ports/unix/main.c

```diff
@@ -592,7 +592,7 @@ MP_NOINLINE int main_(int argc, char **argv) {
 
                 mp_obj_t mod;
                 nlr_buf_t nlr;
-                bool subpkg_tried = false;
+                static bool subpkg_tried = false;
```

Comment:
Probably best to add a comment explaining this because it's non-obvious. And perhaps even to maintain original behavior and always initialize to false eplicitly just in case someone copies this or e.g. calls main multiple times for whatever reason?